docs/configuration.asciidoc

```diff
@@ -79,6 +80,26 @@ NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-transaction-sample-rate]]
+==== `TransactionSampleRate`
+
+By default, the agent will sample every transaction (e.g. request to your service). To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. We still record overall time and the result for unsampled transactions, but no context information, labels, or spans.
+
+NOTE: When parsing value for this option agent doesn't consider the current culture and it expects that dot is used to separate an integer component and a fraction part of a floating-point number.
```

Comment:
Is "dot" a common term in .NET? Changed it to "period" as that's what we use elsewhere in the docs, but feel free to change it back if it makes more sense as dot here. 

```suggestion
NOTE: When parsing the value for this option, the agent doesn't consider the current culture.
It also expects that a period (`.`) is used to separate the integer and the fraction of a floating-point number.
========